src/components/empty-state.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Layer } from '@carbon/react';
+import { Report } from '@carbon/react/icons';
+import styles from './empty-state.scss';
+
+type EmptyState = {
```

Comment:
We don't need a whole component for this. Empty states should match the design of the thing that they are the empty state for. Generic empty states are only useful in contexts where we have many similar widgets.